client/jetpack-cloud/sections/settings/advanced-credentials/credentials-form/index.tsx

```diff
@@ -129,41 +190,47 @@ const ServerCredentialsForm: FunctionComponent< Props > = ( { onCredentialsSave,
 			<div className="credentials-form__row">
 				<FormFieldset className="credentials-form__server-address">
 					<div className="credentials-form__support-info">
-						<FormLabel htmlFor="host-address">{ translate( 'Server Address' ) }</FormLabel>
-						{ hostInfo && hostInfo.serverAddress && (
+						<FormLabel htmlFor="host-address">{ translate( 'Server address' ) }</FormLabel>
+						{ hostInfo?.inline?.serverAddress && (
 							<InfoPopover>
-								<InlineInfo info={ hostInfo.serverAddress } />
+								<InlineInfo
+									credentialType={ formState.protocol }
+									info={ hostInfo.inline.serverAddress }
+								/>
 							</InfoPopover>
 						) }
 					</div>
 					<FormTextInput
 						name="host"
 						id="host-address"
-						placeholder={ translate( 'YourGroovyDomain.com' ) }
-						// value={ get( form, 'host', '' ) }
-						// onChange={ handleFieldChange }
-						// disabled={ formIsSubmitting }
+						placeholder={ translate( 'example.com' ) }
+						value={ formState.host }
+						onChange={ handleFormChange }
+						disabled={ disabled }
 						// isError={ !! formErrors.host }
 					/>
 					{ /* { formErrors.host && <FormInputValidation isError={ true } text={ formErrors.host } /> } */ }
 				</FormFieldset>
 
 				<FormFieldset className="credentials-form__port-number">
 					<div className="credentials-form__support-info">
-						<FormLabel htmlFor="server-port">{ translate( 'Port Number' ) }</FormLabel>
-						{ hostInfo && hostInfo.portNumber && (
+						<FormLabel htmlFor="server-port">{ translate( 'Port number' ) }</FormLabel>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.